hack/protoc.sh

```diff
@@ -39,7 +39,6 @@ generateproto(){
 }
 
 main(){
-    API_ROOT="${DIR}/cri/apis/v1alpha1" YEAR_TIME=" $(date '+%Y')" generateproto
```

Comment:
Why delete it？

Reply:
Now we don't need to extend the API of CRI v1alpha1. :)

Reply:
https://github.com/alibaba/pouch/blob/master/docs/api/GRPC_API_CHANGELOG.md